syntax = "proto2";

package xen;

message ChannelMapping {
    required int32 midiChannel = 1;
    required int32 cvChannelFrom = 2;
    required int32 cvChannelTo = 3;
}

message ChannelConfig {
    repeated ChannelMapping channelMapping = 1;
}

message ConfigWrapper {
    optional ChannelConfig channelConfig = 1;
}
